init-wifi.h

```diff
@@ -31,28 +51,30 @@ void loadWifiConfig(){
   File wifiFile = SPIFFS.open(wifiName);
   if (wifiFile) 
   {
+    Serial.println("Opened file");
     size_t size = wifiFile.size();
     Serial.println("Reading Wifi Config from Flash");
     
-//    std::unique_ptr<char[]> buf(new char[size]);
-//    wifiFile.readBytes(buf.get(), size);
-//    serialStr(buf.get());
+    std::unique_ptr<char[]> buf(new char[size]);
+    wifiFile.readBytes(buf.get(), size);
+    serialStr(buf.get());
     
-    char* b = new char[size];
-    memset(wifiConfig.ssid, '\0', size);
-    wifiFile.readBytes(b, size);
-    serialStr(b);
-
-//    DynamicJsonBuffer jsonBuffer;
-//    JsonObject& json = jsonBuffer.parseObject(buf.get());
+    //char* b = new char[size];
+    //memset(wifiConfig.ssid, '\0', size);
+    //wifiFile.readBytes(b, size);
+    //serialStr(b);
+    //delete[] b;
+
+    DynamicJsonBuffer jsonBuffer;
+    JsonObject& json = jsonBuffer.parseObject(buf.get());
 //    
-//    if (json.success()) {
-//      Serial.println("Wifi Configuration");
-//      json.prettyPrintTo(Serial);
-//      setWifiConfig(json["ssid"], json["pwd"]);
-//    } else {
-//      Serial.println("Wifi Configuration is corrupted");
-//    }
+    if (json.success()) {
+      Serial.println("Wifi Configuration");
+      json.prettyPrintTo(Serial);
+      setWifiConfig(json["wifi_ssid"], json["wifi_pwd"]);
```

Comment:
why are json properties' names changed?

Reply:
Json properties must be same in both functions: saveConfigs and loadConfigs, otherwise we will get infinite module restart loop.
Maybe it is good idea to move properties names to definitions.h and declare as constants.

Reply:
To be honest - I don`t know)
Suggestion is that when we try to fetch with wrong keys - json library returns some garbage.
I can try to rewrite this methods(loadConfigs and storeConfigs) so, that they will not use arduino json  library. 
Is this important now?

Reply:
no
moving properties' names into definitions is pretty enough
btw, I would use arduino json for all of our parse cases - even for parsing data from socket
I have strong opinion we should use same parsing way for all cases - and arduino json parsing fits here. btw, does it support nested properties like {foo: {bar: {zu: 5, flag: false}}}?

Reply:
I'll try that myself
pls only move properties' names to definitions to keep it consistant

Reply:
ok